node_modules/oae-activity/tests/test-email.js

```diff
@@ -855,6 +857,59 @@ describe('Activity Email', function() {
     });
 
     /**
+     * Test that verifies that activity emails are not delivered to users who have deleted their
+     * profiles
+     */
+    it('verify email is not delivered to users who have since deleted their profile', function(callback) {
```

Comment:
Also where would the priority lay with respect to inviting guests? The work is not trivial and I'm assuming needs performance testing.